src/ui/studio/video-setup/prefs.js

```diff
@@ -422,7 +427,7 @@ const RadioButton = ({ id, value, checked, name, onChange, label, state }) => {
         },
       }}
     />
-    <label htmlFor={id}>{ label || value }</label>
+    <label tabIndex={isExpanded ? '0' : '-1'} onKeyDown={e => (e.key === 'Enter' || e.key === ' ' ) && onChange(value)} htmlFor={id}>{ label || value }</label>
```

Comment:
That's a long line, I would break it up.

```suggestion
    <label 
      tabIndex={isExpanded ? '0' : '-1'} 
      onKeyDown={e => (e.key === 'Enter' || e.key === ' ' ) && onChange(value)} 
      htmlFor={id}
    >{ label || value }</label>
```